### PR TITLE
Make references to individual tools pointers to the tools page

### PR DESCRIPTION
**Pull request by scottcwilson**

### CODE REVIEW

**Comment by drbyte** [comment]
Are `SmartFTP` and `CuteFTP` still around? Is there any need to even mention them? Does just ignoring them serve an equal purpose to recommending avoiding them?

**Comment by scottcwilson** [comment]
Both still seem to be available.

File: content/user/first_steps/how_do_i_install.md [hunk 1]
```diff
@@ -30,9 +30,10 @@ If No, stop, and see our [Certified Hosting](http://www.zen-cart.com/hosting) Si
 
 ### 2\. Do you have FTP software?
 
-If No, stop.  You need to obtain a decent FTP software package such as [FileZilla](http://filezilla.sourceforge.net/) to transfer files back and forth from your computer to your webserver (the computer on the internet where you have your domain hosted). 
+If No, stop.  You need to obtain a good [FTP tool](/user/first_steps/useful_tools/#ftp-tools) to transfer files back and forth from your computer to [your web host](/user/first_steps/hosting/#hosting-companies).
 
-<span class="error">**NOTE:** Many people have had timeout and other problems when using programs like SmartFTP and CuteFTP. We recommend that you do NOT use these problematic programs.  
+<span class="error">
+**NOTE:** Many people have had timeout and other problems when using programs like SmartFTP and CuteFTP. We recommend that you do NOT use these problematic programs.  See [FTP tools](/user/first_steps/useful_tools/#ftp-tools) for a list of alternatives.
```
Comment (drbyte) [inline content/user/first_steps/how_do_i_install.md]: Should this `<span>` be closed?

Comment (drbyte) [reply content/user/first_steps/how_do_i_install.md]: Oh, I see, Notes 1 and 2 are nested.

**Pull request merged**